src/public/sass/patterns/_related-items.scss

```diff
@@ -2,11 +2,93 @@
 // can be seen at https://components.publishing.service.gov.uk/component-guide/related_navigation
 
 .app-related-items {
-    border-top: 10px solid govuk-colour("blue");
-    padding-top: govuk-spacing(2);
+  border-top: 10px solid govuk-colour("blue");
+  padding-top: govuk-spacing(2);
+}
+
+.app-related-items .govuk-list > li {
+  margin-bottom: govuk-spacing(2);
+}
+
+.interruption-card {
+  padding: 2em 3em;
+  background-color: govuk-colour("blue");
+
+  &.interruption-card-paper {
+    background-color: govuk-colour("blue");
+  }
+  @include govuk-media-query($until: tablet) {
+    background-image: none !important;
+  }
+
+  .govuk-button {
+    background-color: govuk-colour("white");
+    color: govuk-colour("black");
+
+    &:hover {
+      background-color: govuk-colour("mid-grey");
+    }
+  }
+
+  h1 {
+    color: govuk-colour("white");
   }
-  
-  .app-related-items .govuk-list > li {
-    margin-bottom: govuk-spacing(2);
+
+  .govuk-body,
+  .govuk-body-l {
+    color: govuk-colour("white");
+  }
+
+  ul {
+    color: govuk-colour("white");
+    width: 75%;
+    @include govuk-media-query(mobile) {
+      width: 100%;
+      margin-top: 0.5em;
+    }
+
+    li {
+      h2 {
+        padding-bottom: 0.5em;
+        display: inline-block;
+        width: 85%;
+        margin: 0 0 0 0.2em;
+        @include govuk-media-query(mobile) {
+          width: 80%;
+        }
+      }
+
+      span {
+        vertical-align: top;
+      }
+    }
+  }
+
+  .govuk-link.govuk-link--interrupt {
+    color: govuk-colour("black");
+  }
+
+  .govuk-link {
+    color: govuk-colour("white");
+  }
+
+  .button {
+    background-color: govuk-colour("white");
+    color: $govuk-link-colour;
+    font-weight: 700;
+
+    &:focus,
+    &:hover {
+      background-color: govuk-colour("light-grey");
+      color: $govuk-link-colour;
+    }
+  }
+
+  .circle-white {
+    background-color: govuk-colour("white");
+    color: govuk-colour("black");
+    margin-right: 0.5em;
+    position: relative;
+    top: -0.3em;
   }
-  
\ No newline at end of file
+}
```

Comment:
this file was called _related-items because the original contents were part of the related-items section (or as we were calling it subsection) of the landing page. Im not too bothered about how we distribute things, but we should probably rename this file if we are going to use it for multiple components

Reply:
Check if you like it like this now

Reply:
Looks good! Thanks mate!